tasks/11_find-longest-word/find-longest-word.js

```diff
@@ -0,0 +1,6 @@
+export const findLongestWord = (value) => {
+  return value.split(/\W/g).map((cur) => cur.length).reduce((prev, cur) => {
```

Comment:
it would be better to split by space ` ` or `\s`. you don`t need `g` for reg exp here

you don't need to `map` here, try to use only `reduce`